tests/integration/conftest.py

```diff
@@ -1353,7 +1416,7 @@ def get_qtest_property(model, prop_name):
         custom_props = {}
 
     if prop_name in model_dict:
-        actual_values.append(model[prop_name])
+        actual_values.append(model_dict[prop_name])
```

Comment:
This bug has been here the whole time. Wasn't discovered until I wrote a test that used this pathway.